cluster/examples/kubernetes/ceph/csi/cephfs/storageclass.yaml

```diff
@@ -21,11 +21,11 @@ parameters:
   # The secrets contain Ceph admin credentials. These are generated automatically by the operator
   # in the same namespace as the cluster.
   csi.storage.k8s.io/provisioner-secret-name: rook-csi-cephfs-provisioner
-  csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
+  csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph # namespace:cluster
```

Comment:
These secrets will be with the csi driver, so I believe lines 24-28 should have `# namespace:system`. The same would apply to the other storage class examples.

Reply:
Line 22 suggests they are in the clustern ns

Reply:
Thanks, @Madhu-1 !